lib/nerves_network/if_supervisor.ex

```diff
@@ -37,13 +38,18 @@ defmodule Nerves.Network.IFSupervisor do
     end
   end
 
+  def scan(ifname) when is_atom(ifname), do: scan(to_string(ifname))
```

Comment:
I'd like to start pruning out support for the atom versions of the APIs. Perhaps add a log message to tell people to stop doing this and then sometime later we can make it crash outright.

Reply:
:+1: